app/frontend/src/router/index.js

```diff
@@ -92,7 +92,11 @@ export default function getRouter(basePath = '/') {
             component: () => import(/* webpackChunkName: "submit" */ '@/views/form/Submit.vue'),
             meta: {
               breadcrumbTitle: 'Submit Form',
-              hideNavBar: true
+              hideNavBar: true,
+              hideTitle: true,
+              hideVersionChip: true,
+              dontPrintHeader: true,
+              dontPrintFooter: true
```

Comment:
should stick with "hidePrintHeader" instead of "dontPrintHeader" - better to have the first word be the action or descriptor instead of a negative.

Reply:
Do we need **all** these flags?
What's the exact bussiness case. Is it just that all of these are flagged when the user is filling out a form? Then could it be 1? Or are there different permutations?